data/text_sampling/extend_tabular_processed.py

```diff
@@ -0,0 +1,46 @@
+import glob
+import os
+
+import pandas as pd
+
+if __name__ == "__main__":
+    path_base = __file__.replace("text_sampling/extend_tabular_processed.py", "")
+    path_data_dir = sorted(glob.glob(path_base + "tabular/**/data_clean.csv"))
+    path_processed_smiles = path_base + "text_sampling/extend_tabular_processed.csv"
+
+    cols = [
+        "SMILES",
+        "selfies",
+        "deepsmiles",
+        "canonical",
```

Comment:
Yes, `SMILES` is what comes from the data source, so it could be already in a canonical form, but this is not checked.
Should we change that? We can also look into that when we start with code for SMILES augmentations.

Reply:
no, I guess this is ok as long as we have the canonical form somewhere